Lib/fontParts/base/font.py

```diff
@@ -1537,42 +2100,72 @@ def _isCompatible(self, other, reporter):
     # mapping
     # -------
 
-    def getReverseComponentMapping(self):
-        """
-        Get a reversed map of component references in the font.
-        {
-        'A' : ['Aacute', 'Aring']
-        'acute' : ['Aacute']
-        'ring' : ['Aring']
-        etc.
-        }
+    def getReverseComponentMapping(self) -> ReverseComponentMappingType:
+        """Get a reversed map of all component references in the font.
+
+        This method creates a :class:`dict` mapping each component glyph
+        name in the font to a :class:`set` containing the composite
+        glyph names that include the comoponent.
+
+        Note that one glyph can have multiple unicode values, and a
+        unicode value can have multiple glyphs pointing to it.
+
+        :return: A :class:`dict` of component glyph names mapped to sets
+            of composite glyph names.
+
+        Example::
+
+            {
+            'A': {'Aacute', 'Aring'},
+            'acute': {'Aacute'},
+            'ring': {'Aring'},
+            ...
+            }
+
         """
         return self._getReverseComponentMapping()
 
-    def _getReverseComponentMapping(self):
-        """
+    def _getReverseComponentMapping(self) -> ReverseComponentMappingType:
+        """Get a reversed map of all component references in the font.
+
         This is the environment implementation of
         :meth:`BaseFont.getReverseComponentMapping`.
 
-        Subclasses may override this method.
+        :raises NotImplementedError: If the method has not been
+            overridden by a subclass.
+
+        .. note::
+
+            Subclasses may override this method.
+
         """
         layer = self.defaultLayer
         return layer.getReverseComponentMapping()
 
-    def getCharacterMapping(self):
-        """
-        Create a dictionary of unicode -> [glyphname, ...] mappings.
-        All glyphs are loaded. Note that one glyph can have multiple unicode values,
```

Comment:
Lets not lose the note about multiple unicodes

Reply:
Fixed.